### PR TITLE
Improvements of the Scroll Logo

### PR DESCRIPTION
**Pull request by xHeaveny**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Improvements of the Scroll Logo because the old one was outdated and ugly

### CODE REVIEW

File: resources/contributors.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@ matthewde | https://github.com/m4tthewde | :/avatars/matthewde.jpg | Contributor
 Karar Al-Remahy | https://github.com/KararTY | :/avatars/kararty.png | Contributor
 Talen | https://github.com/talneoran | | Contributor
 SLCH | https://github.com/SLCH | :/avatars/slch.png | Contributor
+xHeaveny | https://github.com/xHeaveny | :/avatars/xheaveny.png | Contributor
```
Comment (Mm2PL) [inline resources/contributors.txt:46]: Adding yourself to contributors should be a separate PR.

File: src/autogenerated/ResourcesAutogen.cpp [hunk 2]
```diff
@@ -62,4 +63,4 @@ Resources2::Resources2()
     this->twitch.vip = QPixmap(":/twitch/vip.png");
 }
 
-}  // namespace chatterino
\ No newline at end of file
+}  // namespace chatterino
```
Comment (Mm2PL) [inline src/autogenerated/ResourcesAutogen.cpp:66]: Looks like your editor is adding trailing new lines at the end.

**Review by Mm2PL — CHANGES_REQUESTED** [review]

File: resources/scrolling/neutrallScroll.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (Mm2PL) [inline resources/scrolling/neutrallScroll.svg:1]: misspelled filename
```diff
-neutrallScroll.svg
+neutralScroll.svg

-neutrallScroll.png
+neutralScroll.png
```

**Review by Mm2PL — CHANGES_REQUESTED** [review]
Still has the unrelated change: adding yourself to contributor list (make another PR after this PR has been merged). You misspelled one filename. I think you might have forgotten to [generateResources](https://github.com/Chatterino/chatterino2/blob/master/resources/generate_resources.py).

File: resources/scrolling/neutrallScroll.svg (hunk 3 above)
Comment (xHeaveny) [reply resources/scrolling/neutrallScroll.svg:1]: Oh thanks, I am German and forgot how to spell `neutral`

File: resources/contributors.txt (hunk 1 above)
Comment (xHeaveny) [reply resources/contributors.txt:46]: So should I take it out?

Comment (Felanbird) [reply resources/contributors.txt:46]: > So should I take it out?

Yes

Comment (xHeaveny) [reply resources/contributors.txt]: Done

**Comment by pajlada** [comment]
Please post a before and after picture

**Comment by xHeaveny** [comment]
> Please post a before and after picture

my compiler wont compile it :c

**Comment by pajlada** [comment]
Doesn't add/change enough of value, so I'm going to go ahead and close this.

Thanks for the contribution anyway!
